datameta/api/openapi.yaml

```diff
@@ -496,19 +496,19 @@ components:
     MetaDataSet:
       type: object
       properties:
-        records:
+        fields:
```

Comment:
On a second thought... what do you think about singular `record`? Then we would have `record.zipCode`, `record.sampleId`,...

Reply:
Whatever you feel most comfortable with. Will change it